src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,99 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.arrayContaining;
+import static org.hamcrest.Matchers.is;
+
+class Exercise1 {
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortLocalComparator() {
+        Person[] persons = getPersons();
+
+        Comparator<Person> personAgeComparator = Comparator.comparingInt(Person::getAge);
+        Arrays.sort(persons, personAgeComparator);
+
+        assertThat(persons, is(arrayContaining(
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        )));
+    }
+
+    @Test
+    void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, Comparator.comparingInt(Person::getAge));
```

Comment:
anonimous comparator  - means you should declare anonimous class by 
new Comparator() {
     public int compare(Object o1, Object o2) {
           return -1;
        }
    });

Reply:
Done.